registries/_format/byte.md

```diff
@@ -1,9 +1,11 @@
 ---
 owner: DarrelMiller
 issue: 
-description: base64 encoded characters
+description: base64 encoded data as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-4)
 base_type: string
 layout: default
+deprecated_note: '3.1'
+remarks: "In OpenAPI 3.1, instead use [`contentEncoding: base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding), optionally alongside [contentMediaType](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentmediatype)."
```

Comment:
This should be `contentEncoding: base64`, as that is the equivalent of `format: byte`.

Reply:
Added suggestion for this.